Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Dashboard/RideBooking/Booking.hs

```diff
@@ -0,0 +1,40 @@
+module Domain.Action.Dashboard.RideBooking.Booking
+  ( postBookingStatus,
+    getBookingList,
+  )
+where
+
+import qualified "this" Domain.Action.UI.Booking
+import qualified "this" Domain.Types.Booking
+import qualified "this" Domain.Types.Booking.API
+import qualified Domain.Types.Merchant
+import qualified "this" Domain.Types.Person
+import qualified Environment
+import EulerHS.Prelude hiding (id)
+import qualified Kernel.Prelude
+import qualified Kernel.Types.Beckn.Context
+import qualified Kernel.Types.Id
+import SharedLogic.Merchant (findMerchantByShortId)
+
+postBookingStatus ::
+  Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant ->
+  Kernel.Types.Beckn.Context.City ->
+  Kernel.Types.Id.Id Domain.Types.Booking.Booking ->
+  Kernel.Types.Id.Id Domain.Types.Person.Person ->
+  Environment.Flow Domain.Types.Booking.API.BookingAPIEntity
+postBookingStatus merchantShortId _opCity bookingId personId = do
+  m <- findMerchantByShortId merchantShortId
+  Domain.Action.UI.Booking.bookingStatus bookingId (personId, m.id)
```

Comment:
earlier it was:

```
callBookingStatus :: ShortId DM.Merchant -> Id SRB.Booking -> Id DP.Person -> FlowHandler DB.BookingAPIEntity
callBookingStatus merchantId bookingId personId = do
  m <- withDashboardFlowHandlerAPI $ findMerchantByShortId merchantId
  UB.bookingStatus bookingId (personId, m.id)

bookingStatus :: Id SRB.Booking -> (Id Person.Person, Id Merchant.Merchant) -> FlowHandler BookingAPIEntity
bookingStatus bookingId = withFlowHandlerAPI . DBooking.bookingStatus bookingId
```
so `withFlowHandlerAPI` wrapper changed to `withDashboardFlowHandlerAPI` in all `RideBooking` apis